model/src/main/java/org/cbioportal/model/DataAccessToken.java

```diff
@@ -1,22 +1,36 @@
 package org.cbioportal.model;
 
 import java.io.Serializable;
+import java.util.Date;
 import javax.validation.constraints.NotNull;
 
 public class DataAccessToken implements Serializable {
 
     private String token;
-
-    public DataAccessToken(String token) {
```

Comment:
having the constructor defined in the class was breaking the auto-mapping behavior when using mybatis. Planning to look into this more.